lib/misc.c

```diff
@@ -198,7 +202,12 @@ void strip_html(char *in)
 			}
 		} else if (*in == '&') {
 			cs = ++in;
-			while (*in && g_ascii_isalpha(*in)) {
```

Comment:
Isn't it simpler to just replace g_ascii_isalpha with g_ascii_isalnum here?

Reply:
That is what I did, but I don't know if `#` is considered numeric

Reply:
It wouldn't be, I meant g_ascii_isalnum() || *in == '#' here, but this works too.